ward/suite.py

```diff
@@ -15,9 +15,16 @@ class Suite:
     cache: FixtureCache = field(default_factory=FixtureCache)
 
     @property
-    def num_tests(self):
+    def num_tests(self) -> int:
         return len(self.tests)
 
+    @property
+    def num_tests_with_parameterization(self) -> int:
```

Comment:
Could you spell this with an `s` instead of `z` (UK spelling) to keep things consistent?

Reply:
> Could you spell this with an `s` instead of `z` (UK spelling) to keep things consistent?

As long as we have `e` between the `t` and the `r`, I'm happy :)